### DISCUSSION TITLE
CryEngine

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#65] [General]
# CryEngine

Check if CryEngine is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/65a85b5a-ab16-4f80-ba1f-b551dc958588

**Comment by SolberLight** [comment]
No it's not then. It's just emulation not native arm build

**Comment by s-a-n-n-i** [comment]
Hi, I've recently tried to download CryEngine 5.7 LTS and it failed. Do you know how to resolve this issue? Thanks for your help.
